sssom/util.py

```diff
@@ -832,7 +828,7 @@ def inject_metadata_into_df(msdf: MappingSetDataFrame) -> MappingSetDataFrame:
     :return: MappingSetDataFrame with metadata as columns
     """
     # TODO Check if 'k' is a valid 'slot' for 'mapping' [sssom.yaml]
-    with open(SCHEMA_YAML) as file:
+    with open(SSSOMSchemaView().yaml) as file:
         schema = yaml.safe_load(file)
     slots = schema["classes"]["mapping"]["slots"]
```

Comment:
seems redundant now.. Shouldnt this simply be part of the new `SSSOMSchemaView` class object?

Reply:
Why would you load the yaml again, rather than using `SSSOMSchemaView.dict`?

Reply:
Good catch!